external/broker/broker.go

```
@@ -3,38 +3,44 @@ package broker
 	return &broker{
 		brokerSyncMap: brokerSyncMap,
 	}
 }
 
-func (b *broker) getBrokerService(ctx context.Context, brokerId int64) (Broker, error) {
-	if broker, ok := b.brokerSyncMap.Load(brokerId); ok {
-		return broker.(Broker), nil
+type BrokerService interface {
+}
+
+func (b *broker) getBrokerService(ctx context.Context, brokerType brokerType) (Broker, error) {
```

golangci-lint (1.62.2)

func `(*broker).getBrokerService` is unused

(unused)